presto-main/src/test/java/com/facebook/presto/sql/planner/TestPlanMatchingFramework.java

```diff
@@ -69,6 +73,24 @@ public void testOutputSameColumnMultipleTimesWithOtherOutputs()
     }
 
     @Test
+    public void testStrictOutput()
+    {
+        assertMinimallyOptimizedPlan("SELECT orderkey, extendedprice FROM lineitem",
+                strictOutput(ImmutableList.of("ORDERKEY", "EXTENDEDPRICE"),
+                        tableScan("lineitem", ImmutableMap.of("ORDERKEY", "orderkey",
```

Comment:
Can you also use `strictOutput` after intermediate nodes after `tableScale`? Let's say I want to test strict output of pipeline:
`project -> filter -> tableScan`, where `project` output symbols should be strictly symbols from `tableScan`?